bundles/org.eclipse.equinox.frameworkadmin.test/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %Bundle-Name
 Bundle-SymbolicName: org.eclipse.equinox.frameworkadmin.test
-Bundle-Version: 1.3.0.qualifier
+Bundle-Version: 1.3.100.qualifier
```

Comment:
build.properties should be part of the jgit.ignore in the pom.xml here, because changing it doesn't affect the bundle and should require to bump the version.

Reply:
> because changing it doesn't affect the bundle and should require to bump the version.

It depends, you can configure included content and that does affect the bundle, also this change affects the bundle (even though in a way that should be equivalent to the previous state)

Reply:
Absolutely, we can't ignore build.properties due to bin.includes defined in it.